src/drivers/distance_sensor/ll40ls/LidarLiteI2C.h

```diff
@@ -66,6 +67,8 @@ static constexpr uint8_t LL40LS_UNIT_ID_LOW	= 0x17;
 
 static constexpr uint8_t LL40LS_SIG_COUNT_VAL_REG	= 0x02;	/* Maximum acquisition count register */
 static constexpr uint8_t LL40LS_SIG_COUNT_VAL_MAX	= 0xFF;	/* Maximum acquisition count max value */
+static constexpr int LL40LS_SIGNAL_STRENGH_MIN_V3HP = 70;	// Min signal strength for V3HP
```

Comment:
@cmic0 can you match the comment format with the above? Thanks

Reply:
Also, missing a `T`:
```suggestion
static constexpr int LL40LS_SIGNAL_STRENGTH_MIN_V3HP = 70;	// Min signal strength for V3HP
```